tcmu.conf

```diff
@@ -15,4 +15,4 @@
 # Logging Directory Path
 # The default logging Directory path is /var/log, uncomment it
 # and set your own path:
-# log_dir_path = "/var/log"
+# log_dir = "/var/log"
```

Comment:
What was the reason for the rename? I just saw the mention in the git commit and not a reason.

Normally I would not care. I like log_dir more, but because it is a user facing value it is a concern.

Reply:
Frankly, the naming looks a bit awkward to me.

We need to support replacing the tcmu.conf (just in case if new variables are introduced or defaults are changed it will be helpful. *Note* there was a recent change to tcmu.conf about introducing CRIT loglevel already)

As we anyway need to support replacing this file, It is fine to tune the option name here, as the new option name is presented to user anyway. We still save one old tcmu.conf as backup after replace (we can increase to hold more later)